libkbfs/folder_branch_ops.go

```diff
@@ -1444,8 +1444,27 @@ func (fbo *folderBranchOps) execMDReadNoIdentifyThenMDWrite(
 	return err
 }
 
+func (fbo *folderBranchOps) makeEntryInfoExtendedMap(ctx context.Context,
+	eis map[string]EntryInfo) map[string]EntryInfoExtended {
+	ret := make(map[string]EntryInfoExtended)
+	for k, v := range eis {
+		ret[k] = fbo.makeEntryInfoExtended(ctx, v)
+	}
+	return ret
+}
+
+func (fbo *folderBranchOps) makeEntryInfoExtended(ctx context.Context,
+	ei EntryInfo) EntryInfoExtended {
+	eie := EntryInfoExtended{EntryInfo: ei}
+	_, uid, err := fbo.config.KBPKI().GetCurrentUserInfo(ctx)
+	if err == nil { // writable is only possible if user is logged in
```

Comment:
This hides any other legit errors, better to use `GetCurrentUsernameIfPossible`.